mock/Cargo.toml

```diff
@@ -39,5 +39,5 @@ default_features = false
 features = ["native-tls"]
 
 [features]
-nightly-testing = ["clippy", "rusoto_core/nightly-testing"]
+nightly-testing = ["rusoto_core/nightly-testing"]
```

Comment:
Was going to remove the nightly-testing feature as well since it all just boils down to clippy but figured there might be future nightly features so no harm in keeping